src/josepy/jwk_test.py

```diff
@@ -320,5 +323,102 @@ def test_encode_y_leading_zero_p256(self):
         JWK.from_json(data)
 
 
+class JWKOKPTest(unittest.TestCase):
+    """Tests for josepy.jwk.JWKOKP."""
+    # pylint: disable=too-many-instance-attributes
+
+    # TODO: write the thumbprint
+    thumbprint = (
+        b'kPrK_qmxVWaYVA9wwBF6Iuo3vVzz7TxHCTwXBygrS4k'
```

Comment:
I'm not sure what the thumbprint should be here. Some guidance would be appreciated.

Reply:
@bmw @ohemorange Any help on this?

Reply:
I think this is fine. I'll try to push some more tests, and changes. I'm not really sure why the thumbprint of all of the tested keys is not tested, please see `JWKECTest` for an example. So I think just testing one key works?

Reply:
I think a gist to generate them would be helpful.

Reply:
Ah I see. Yes, I think testing all of them would be ideal.

I think there could be a clever design here either extending base class logic and giving each key its own test class or having a list of thumbprints and keys to test. If testing all of them proves difficult for some reason though, I wouldn't worry too much about it.